src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java

```diff
@@ -0,0 +1,93 @@
+package teammates.test.cases.ui.browsertests;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.util.Const;
+import teammates.common.util.ThreadHelper;
+import teammates.common.util.Url;
+import teammates.test.pageobjects.Browser;
+import teammates.test.pageobjects.BrowserPool;
+import teammates.test.pageobjects.InstructorFeedbackEditPage;
+import teammates.test.util.Priority;
+
+
+@Priority(-1)
+public class InstructorFeedbackEditCopyUiTest extends BaseUiTestCase {
+    private static Browser browser;
+    private static InstructorFeedbackEditPage feedbackEditPage;
+    private static DataBundle testData;
+    private static String instructorId;
+    private static String courseId;
+    private static String feedbackSessionName;
+
+    @BeforeClass
+    public static void classSetup() throws Exception {
+        printTestClassHeader();
+        testData = loadDataBundle("/InstructorFeedbackEditCopyTest.json");
+        removeAndRestoreTestDataOnServer(testData);
+        instructorId = testData.accounts.get("instructorWithSessions").googleId;
+        courseId = testData.courses.get("course").id;
+        feedbackSessionName = testData.feedbackSessions.get("openSession").feedbackSessionName;
+
+        browser = BrowserPool.getBrowser();
+    }
+
+    @Test
+    public void allTests() throws Exception{
+        feedbackEditPage = getFeedbackEditPage();
+        
+        ______TS("Submit empty course list");
+        feedbackEditPage.clickFsCopyButton();
+        ThreadHelper.waitFor(1000);
```

Comment:
If possible, wait for element to appear instead of waiting for an absolute time as it will be more reliable.